pypeclub/OpenPype
# Ftrack: Add more options for note text of integrate ftrack note

## Brief description
Added option to fill published files and host name in note during integration of ftrack note.

## Description
Reorganized integrate ftrack api plugin to be more readable. Integrate Ftrack Api now does not collect only used asset versions but also component data that were used to create them which gives ability to get published paths.

Integrate ftrack note does not require comment from artist to be able add the note, because the note may consist only from published paths and intent, but empty notes are not created. Added 4 new possible keys into note template `{host_name}`, `{app_name}`, `{app_label}` and `{published_paths}` which adds all published paths.

Extended host name collector to collect app name and app label into context.

## Additional info
Changed key in settings because note template was extended.
From:
`project_settings/ftrack/publish/IntegrateFtrackNote/note_with_intent_template`
Into:
`project_settings/ftrack/publish/IntegrateFtrackNote/note_template`

## Testing notes:
1. Enable `IntegrateFtrackNote` and change the format to contain new keys
2. Publish something which will publish to ftrack
3. Check the note on published asset version
